qemu/tests/cfg/numa_memdev_options.cfg

```diff
@@ -0,0 +1,49 @@
+- numa_memdev_options:
+    type = numa_memdev_options
+    kill_vm_on_error = yes
+    numa_hardware_cmd = "numactl --hardware"
+    mem_fixed = 1536
+    mem_devs = "mem0 mem1"
+    backend_mem = memory-backend-ram
+    use_mem_mem0 = no
+    use_mem_mem1 = no
+    host-nodes_mem0 = 0
```

Comment:
Any reason to use both "_" and "-" for a variant?

Reply:
This is not an arbitrary notation. "host-nodes" is the fixed key, with "_" and memory device name (mem0/mem1) after it will specify the host-nodes attribute value of that memory device. Thanks!